common/config/rush/pnpm-lock.yaml

```diff
@@ -96,6 +96,28 @@ packages:
       node: '>=8.0.0'
     resolution:
       integrity: sha512-xxlGwbbTkEZAline8wrP75FLaOsT23nMukzIg5X/Cs6cQSQ/JKj7Uxq5Idxp9GgAYT+g+PAj+BJ929/jaselBQ==
+  /@azure/core-amqp/2.0.0:
```

Comment:
It is weird that this file is getting changed, usually this does not happen for simple version updates right?

Reply:
This file is changing because we made event-hubs be a beta version, but checkpointstore depends on the latest, so we end up with some changes here now.